src/Microsoft.Security.Utilities.Core/HighPerformanceScanner.cs

```diff
@@ -156,7 +162,7 @@ private static bool Match(CompiledHighPerformancePattern pattern,
     /// The performance of this method is highly sensitive to the version of
     /// .NET that is used.
```

Comment:
Related to previous comment, sorry I can't comment directly on the other code location. We can see in the Match implementation that we farm regex processing to an engine that professes to return multiple matches, if present, but we only return the first one. So again, some kind of code discussion on this may be helpful.

Reply:
Actually, it wouldn't run off because we currently enforce anchoring to the beginning but we may end up relaxing that.

Reply:
Makes sense, and obviously helpful to document in source.